src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -264,8 +284,8 @@ class SqlKernel extends Disposable implements nb.IKernel {
 			this._queryRunners.set(queryRunnerUri, queryRunner);
 			this.queryManagementService.registerRunner(queryRunner, queryRunnerUri);
 			this._connectionManagementService.connect(this._currentConnectionProfile, queryRunnerUri).then((result) => {
-				this.addQueryEventListeners(queryRunner);
-				queryRunner.runQuery(code).catch(err => onUnexpectedError(err));
+				this.addQueryEventListeners(<QueryRunner>queryRunner);
```

Comment:
Why is the cast needed?

Reply:
There was an error since queryRunner's type is QueryRunner | undefined. But we know here that queryRunner is defined.

Reply:
Use ! for that then 

`this.addQueryEventListeners(queryRunner!);`